app/design/frontend/Skin/default/etc/view.xml

```diff
@@ -0,0 +1,315 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<view xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Do you really need this file? Seems to me that it should be inherited from parent theme